Booklist #Issue222

sohini3062 opened the pull request:
# Description
Added a suggested books list webpage under More on the navigation bar using HTML and CSS
Please include a summary of the change and which issue is fixed.
Fixes #222 
## List any dependencies that are required for this change 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before :
    - After :
![1](https://user-images.githubusercontent.com/64258756/111906155-392d2a00-8a75-11eb-9cd5-67faad1707da.JPG)
![2](https://user-images.githubusercontent.com/64258756/111906167-48ac7300-8a75-11eb-913b-25cede75a105.jpg)


## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

harikesh409 commented on index.html:
```diff
@@ -75,16 +75,15 @@
                         <a class="nav-link  font-weight-bold nav-items" href="#footer">Contact</a>
                     </li>
-                    <li class="nav-item dropdown">
+
```
Any particular reason to remove it?

harikesh409 replied:
Add a link in the more section pointing to the booklist.html so that the user can navigate it from the homepage.

harikesh409 commented on bookliststyle.css:
```diff
@@ -0,0 +1,142 @@
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  color:white;
+  background-image: url("6.jpg");
```
include the jpg file in the commit and give it a meaningful name.

harikesh409 replied:
place that image in the images folder.

harikesh409 commented on bookliststyle.css:
```diff
@@ -0,0 +1,142 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto+Mono|Roboto:400,700&display=swap");
```
keep the whole file in the css folder.

harikesh409 commented on booklist.html:
```diff
@@ -0,0 +1,39 @@
+    <link rel="stylesheet" href="bookliststyle.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <script src=""></script>
```
unnecessary blank script tag remove it.

harikesh409 commented on booklist.html:
```diff
@@ -0,0 +1,39 @@
+    <script src=""></script>
+</head>
+
+<body>
```
use the layout of the index.html instead of rewriting the whole styles and include the header and footer also.

sohini3062 replied:
Okay I am working on it.

harikesh409 replied:
still it has its own layout instead of reusing the styles.

sohini3062 commented:
I have made some changes via new commits which includes adding footer, giving meaningful name to the background image and updating the css files and image files. I am not sure about updating the header. 
Can you check if these files have been uploaded or I need to make another pull request?
![1](https://user-images.githubusercontent.com/64258756/112149489-0ade5480-8c05-11eb-983d-0ff7f5757fb7.jpg)
![2](https://user-images.githubusercontent.com/64258756/112149499-0f0a7200-8c05-11eb-8314-15181bd48227.JPG)
![3](https://user-images.githubusercontent.com/64258756/112149502-103b9f00-8c05-11eb-8976-5d54d31f41fa.jpg)

sohini3062 commented:
Is there any problem with the new commits or any other changes required? If there are please do let me know, I will work on it.

harikesh409 commented on css/bookliststyle.css:
```diff
@@ -0,0 +1,205 @@
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  color:white;
+  background-image: url("./images/book_background.jpg");
```
the url is wrong check it.

harikesh409 reviewed (changes requested):
The footer is missing few social icons that are present on the main page.

sohini3062 commented:
Okay I am working on them.

sohini3062 commented:
I have updated and added the new footer social icons
![x1](https://user-images.githubusercontent.com/64258756/112850136-5db58180-90c7-11eb-96a2-b10514e80a59.JPG)
I am reusing the styles as in index.html 
![x2](https://user-images.githubusercontent.com/64258756/112850396-95242e00-90c7-11eb-8d3f-2bf5a4596e33.JPG)
I have updated the URL
![x3](https://user-images.githubusercontent.com/64258756/112850509-b422c000-90c7-11eb-8dbc-b37f20edc471.JPG)

Please let me know if there are any mistakes. I am new to this and hence I am making a lot of mistakes. Hope to get better at this soon. Thank you for all your help.

harikesh409 commented on css/bookliststyle.css:
```diff
@@ -0,0 +1,205 @@
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  color:white;
+  background-image: url("images/book_background.jpg");
```
The url should be `../images/book_background.jpg`

sohini3062 replied:
Thank you. Updated it

harikesh409 commented on booklist.html:
```diff
@@ -0,0 +1,136 @@
+            </a>
+          </li>
+          <li class="list--item">
+            <a href="javascript:void(0)" class="list--link">
```
If possible keep online pdf links or amazon links.

sohini3062 replied:
Added Amazon Links

harikesh409 reviewed (approved):
Pull the latest code and merge your changes into it as the new changes are missing.

sohini3062 commented:
I have made a new pull request and merged it. Can you check once?

harikesh409 commented:
> I have made a new pull request and merged it. Can you check once?

What is the PR id?

sohini3062 commented:
Pr id is #306

harikesh409 commented:
Moved to #306
